.github/workflows/sync_board.yml

```
@@ -0,0 +1,56 @@
+        with:
+          python-version: '3.x'
+      - name: Generate board
+        run: |
+          python scripts/generate_tasks_board.py
+      - name: Commit board
+        uses: stefanzweifel/git-auto-commit-action@v5
+        with:
+          commit_message: 'chore(collab): update project board'
+          file_pattern: collaboration/board.md
+      - name: Post board to Discord (optional)
+        if: ${{ secrets.DISCORD_WEBHOOK_URL != '' || secrets.DISCORD_PM_WEBHOOK_URL != '' }}
```

actionlint (1.7.7)

context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)